spec/features/anonymous_user_happy_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+feature 'Anonymous User visits the site and enters an ugly URL' do
+  scenario 'they get back a beautiful one', js: true do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.